classes/book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './item'
+
+class Book < Item
+  attr_accessor :title, :publisher, :cover_state
+
```

Comment:
### OPTIONAL

- [ ] For best practice, if you don't use attr_writer just use attr_reader. I see you mostly use attr_accessor in all classes.

Reply:
Thank you